### PR TITLE
This PR adds recipes for epics-striptool.

### PR DESCRIPTION
**Pull request by pthomasj**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by pthomasj** [comment]
@conda-forge/staged-recipes This recipe is ready for review. Thank you.

**Comment by duncanmmacleod** [comment]
I'm happy to be named as a maintainer of this recipe.

**Comment by pthomasj** [comment]
@conda-forge-admin, please ping team

File: recipes/epics-striptool/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export MOTIF_INC="${PREFIX}/include"
+export MOTIF_LIB="${PREFIX}/lib"
+export X11_INC="${PREFIX}/include"
+export X11_LIB="${PREFIX}/lib"
+
+_make="make -j1 V=1 VERBOSE=1 EPICS_BASE=${EPICS_BASE} TOP=${TOP}"
+
+# build
+${_make} build
+
```
Comment (ocefpaf) [inline recipes/epics-striptool/build.sh:15]: Is there a make check/test we can run here?

Comment (duncanmmacleod) [reply recipes/epics-striptool/build.sh:15]: @ocefpaf, unfortunately not: https://github.com/epics-extensions/StripTool/blob/StripTool-2-5-17-0/Makefile.

File: recipes/epics-striptool/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - xorg-libxt  # [unix]
+    - xorg-libxpm  # [unix]
+
+test:
+  commands:
+    - test -x ${PREFIX}/epics/bin/${EPICS_HOST_ARCH}/StripTool
+
+about:
+  home: https://epics.anl.gov/extensions/StripTool/index.php
+  license: EPICS
```
Comment (jakirkham) [inline recipes/epics-striptool/meta.yaml:45]: Is there a license file we can include in the package?

Comment (ocefpaf) [reply recipes/epics-striptool/meta.yaml:45]: The linter failed here silently here :-/

Comment (jakirkham) [reply recipes/epics-striptool/meta.yaml:45]: Filed a feedstock issue ( https://github.com/conda-forge/epics-striptool-feedstock/issues/1 )

Comment (pthomasj) [reply recipes/epics-striptool/meta.yaml:45]: I sent an email to the EPICS collaboration to see if a license file can be added to the source.

**Pull request merged**